iOSLearningProject/iOSLearningProject/UIComponents/TabBar/MapViewController.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  MapViewController.swift
+//  UIOne
+//
+//  Created by Srushti Sarju Engineer on 07/03/25.
+//
+
+import UIKit
+import MapKit
+
+class MapViewController: UIViewController {
+
+    var locationManager = CLLocationManager()
+    var initialLocation = CLLocation(latitude: 23.027761414755965, longitude: 72.50702246370568)
+    
+    @IBOutlet weak var mapView: MKMapView!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        checkDeviceLocationPermission()
+        centerLocation()
+        cameraBoundary()
+        addAnnotation()
+    }
+    
+//    MARK: - Methods
```

Comment:
Marks should not be applied like this

it should be aligned with the reset of the code